src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -50,13 +55,30 @@ public static TryCatchStatement composeRpcMethodSampleCode(
 
   private static TryCatchStatement composeUnaryRpcMethodSampleCode(
       Method method, List<MethodArgument> arguments, TypeNode clientType) {
-    // TODO(summerji): compose sample code for unary rpc method.
+    // Assign each method arguments with default value.
+    List<Statement> bodyStatements =
+        arguments.stream()
+            .map(
+                methodArg ->
+                    ExprStatement.withExpr(assignMethodArgumentWithDefaultValue(methodArg)))
+            .collect(Collectors.toList());
+    // Invoke current method based on return type.
+    // e.g. if return void, echoClient.echo(..); or,
+    // e.g. if return other type, EchoResponse response = echoClient.echo(...);
+    Expr invokeMethodExpr =
+        method.outputType().equals(TypeNode.VOID)
+            ? MethodInvocationExpr.builder()
+                .setExprReferenceExpr(createVariableDeclExpr(getClientName(clientType), clientType))
+                .setMethodName(method.name())
+                .setReturnType(clientType)
+                .build()
+            : createAssignExprForVariableWithClientMethod(
+                RESPONSE_VAR_NAME, method.outputType(), clientType, method.name(), arguments);
+    bodyStatements.add(ExprStatement.withExpr(invokeMethodExpr));
+
```

Comment:
Could you please confirm if more sample code will be coming?

Reply:
Sorry, I am not sure if I am understanding correctly.

This PR is aim to produce the sample code for pure unary Rpc method kind. There are more coming for other kinds  methods:
- paged unary rpc method
- lro unary rpc method
- unary default rpc method
and rpc callable methods where includes:
- stream rpc method
- paged rpc callable method
- lro rpc callable method
- unary callable method